python/cudf/cudf/utils/utils.py

```diff
@@ -265,6 +265,60 @@ def compare_and_get_name(a, b):
     return None
 
 
+def hash_cudf_object(
```

Comment:
This looks like it should just use [`gdf_hash` ](https://github.com/rapidsai/cudf/blob/branch-0.9/cpp/include/cudf/functions.h#L484) or am I misunderstanding what the goal is here?

Reply:
https://github.com/rapidsai/cudf/blob/049579c7f070172b2c24a3e78256d87c2737f5ba/python/cudf/cudf/dataframe/numerical.py#L466

This is the corresponding Python bindings.

Reply:
Thanks for this pointer @jrhemstad. Synced up with Keith on this will re-work on this.